content/src/content/jcr_root/apps/core/email/components/teaser/v1/teaser/readme.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2021 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Teaser Component
+
+Extensible teaser component for the Core Email Components for composing campaign content written in HTL and based on the [Teaser Core Component](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/teaser/v2/teaser)
+
+The Teaser Component serves as a combination of title, text, image, and call-to-action.
+
+## Features
+
+* Allows the creation of different containers for each segment and its content
+* Allowed components can be configured through policy configuration.
+* Navigation between container panels via the toolbar
+
+## Technical Details
+
+This component inherit its structure and behavior from the [Teaser component (v2)](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/teaser/v2/teaser) of the core.wcm.component package. The changes are the integration of the personalization plugin, which adds a button to insert Adobe Campaign personalization fields in the following locations:
+
+* Configuration dialog toolbar
+* Full screen configuration dialog toolbar
+
+The Teaser also offers the capability to display the image as a background image or as a regular image.
```

Comment:
How does this work? In the UI, there is no indication of this feature. If it is considered "advanced" or only for developers, we need to document it more clearly here. If It is expected that the author user can use this feature, we will need to include it in the Experience League docs as well.

Reply:
removed